### PR TITLE
Feature/my holidays

### PR DESCRIPTION
**Pull request by Kacym**

### CODE REVIEW

File: src/components/MyHolidays.jsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                     switch (event.target.innerText) {
+                        case 'Редактировать':
+                           setPreview({
+                              file: null,
+                              url: holiday.image,
+                           })
+                           openAndCloseHolidayModalHandler(
+                              {
+                                 nameHoliday: holiday.nameHoliday,
+                                 // image: holiday.image,
```
Comment (NuraiymMamatova) [inline src/components/MyHolidays.jsx]: Please delete this comment.

File: src/store/holiday/holdiaySlice.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { getAllHolidaysByUserId, getAllWishesByHolidayId } from './holidayThunk'
+
+const initialState = {
+   // nameOfHoliday: null,
```
Comment (NuraiymMamatova) [inline src/store/holiday/holdiaySlice.js]: Please delete these comments also.

File: src/store/holiday/holidayThunk.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+export const getAllWishesByHolidayId = createAsyncThunk(
+   'holiday/getAllWishesByHolidayId',
+   async (holidayId, { rejectWithValue }) => {
+      try {
+         const response = await axiosInstance.get(
+            `/holidays/getAllWish/${holidayId}`
+         )
+         return response.data
+      } catch (error) {
+         // TODO: toast without promise error
```
Comment (NuraiymMamatova) [inline src/store/holiday/holidayThunk.js]: Please add toastWithoutPromise for error.

Comment (NuraiymMamatova) [reply src/store/holiday/holidayThunk.js]: And after it delete this comment.

File: src/components/HolidayInnerPage.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            <Card
+               variant="secondary"
+               cardName={wish.wishName}
+               cardImage={wish.wishImage}
+               date={convertDateFormat(wish.dateOfHoliday)}
+               holiday={wish.holidayName}
+               status={wish.wishStatus}
+               showBottomBooker="true"
+               // TODO
+               //    meatballsOptions={}
```
Comment (Satybaldyeva) [inline src/components/HolidayInnerPage.jsx]: why didn't you pass the meatballsOptions and handle change ?

Comment (NuraiymMamatova) [reply src/components/HolidayInnerPage.jsx]: Ariet will do it. These meatballs and their methods are already implemented by him. He'll just connect it.

Comment (Satybaldyeva) [reply src/components/HolidayInnerPage.jsx]: aaaaa ok

**Review by Satybaldyeva — APPROVED** [review]

**Pull request merged**